app/Http/Controllers/PlannerController.php

```diff
@@ -131,9 +136,9 @@ public function validateData()
         return request()->validate([
             'festival_id' => 'required',
             'info_text' => 'required|min:3',
-            'planner_image' => '',
-            // 'todo_list' => '',
-            // 'playlists' => '',
+            'todo_list' => '',
```

Comment:
I don't know what exactly is not working, but you could try adding 'nullable' when validating optional fields